util/topsql/stmtstats/aggregator.go

```diff
@@ -19,9 +19,12 @@ import (
 	"sync"
 	"time"
 
+	"github.com/pingcap/tidb/util/topsql/state"
 	"go.uber.org/atomic"
 )
 
+const maxStmtStatsSize = 1000000
```

Comment:
@crazycs520 Please help me to confirm whether this value is enough to protect TiDB.